tests/rptest/tests/rpk_topic_test.py

```diff
@@ -288,9 +288,9 @@ def cond():
     @cluster(num_nodes=4)
     def test_analyze(self):
         topic = "test_analyze_topic"
-        partitions = 25
+        partitions = 1
         runtime_s = 15
-        throughput_bytes_s = 25 * 1024  # 25 KiB/s
+        throughput_bytes_s = 10 * 1024  # 10 KiB/s
```

Comment:
What is the current failure mode. `rpk analyze` times out?

Isn't it a big problem rpk analyze can't even handle 10 KiB/s throughput (which is microscopic) even in docker?